atmos_held_suarez_obs/create_artifact.jl

```diff
@@ -0,0 +1,46 @@
+using ClimaArtifactsHelper
+using ClimaAnalysis
+using Statistics
+import JLD2
+
+# Observation map: Take 60-day zonal average of air temperature at z = 242m
+function process_member_data(simdir)
+    ta = get(simdir; short_name = "ta", reduction = "average", period = "60d")
+    area_avg_ta_second_height = slice(average_lat(average_lon(ta)), z = 242)
+    observation =
+        Float64(slice(area_avg_ta_second_height, time = 2.0736e7).data[1])
+    variance = Matrix{Float64}(undef, 1, 1)
+    variance[1] = var(area_avg_ta_second_height.data)
+    return (; observation, variance)
+end
+
+const output_dir = basename(@__DIR__) * "_artifact"
+const FILE_PATHS = (restart_file, "obs_mean.jld2", "obs_noise_cov.jld2")
+const restart_file = "day200.0.hdf5"
+const model_output_dir = joinpath(output_dir, "truth_simulation", "output_active")
+if isdir(output_dir)
+    @warn "$output_dir already exists. Content will end up in the artifact and may be overwritten."
+    @warn "Abort this calculation, unless you know what you are doing."
+else
+    mkdir(output_dir)
+end
+
+if !all(isfile.(FILE_PATHS))
+    @info "Artifact not found in $output_dir, running ClimaAtmos to generate it. This will take ~3 hours."
+    run(`sbatch generate_observations.sbatch`)
```

Comment:
This will only work on the Caltech cluster. I would suggest you move the code to a separate script and have the sbatch script call that script. You can include the script here so that anyone can run this code on their machine of choice. Add plenty of instructions.

Reply:
Thanks for the comments, I am addressing them. 

I am a bit confused about the requirements of create_artifacts.jl. Is it acceptable to assume that before running `create_artifacts.jl`, the user will run ClimaAtmos to obtain the necessary diagnostic output if I give them a instructions and a script to do it?

Reply:
There are no strict requirements.The guiding principle is: we want this to be clear and reproducible by everyone. 

It is perfectly fine to instruct the user to do something else before running create_artifacts.jl, as long as everything is documented and reproducible.